src/components/DropperClaimsListItem.tsx

```diff
@@ -63,12 +78,20 @@ const DropperClaimsListItem = ({
       setShow(true)
       return
     }
+    if (lFilter === "inactive" && dbData?.active === false) {
```

Comment:
`lFilter` is not a descriptive variable name for me.

Similarly `dbData`.

Would be good if the variable names described the kind of data they held. Would make it easier for me to understand this code.